entc/integration/ulid/ent/schema/mixin/id_mixin.go

```diff
@@ -0,0 +1,18 @@
+package mixin
+
+import (
+	"entgo.io/ent"
+	"entgo.io/ent/schema/field"
+	"entgo.io/ent/schema/mixin"
+	"github.com/oklog/ulid/v2"
```

Comment:
A simpler solution for testing it and also to avoid creating a new module and a directory is to create a dummy package under `integration/ent/schema/*` and use it there. It will also allow us to test it without downloading `oklog/ulid` as a dependency. WDYT?

Reply:
I started with that, I created a custom package only to be imported by the test, but after while I thought that it might be better to use the `ulid` package since it was what the user wanted and it would work as an example of using a compatible uuid type/`ulid`.

But I don't really have a opinion about it, I guess it's better to avoid having to download it only to run the tests.

I'm not sure about the part of avoiding creating a new module, what I want to test is packages that use version like `v2` on the import path, I think I would need to create a new module and if I create a new module, it will be added to the upper `go.mod`, so I have to create yet another module on the "local test".

Or maybe I'm missing something.

Example using local packages (to compare if its better or I can do something different):
- https://github.com/crossworth/ent/tree/import-bug-a/entc/integration/packagewithalias

Let me know what you think.

Reply:
Hey @crossworth! I didn't have enough time to dig into this but would love to have this fix added. Regarding the tests, we already have this logic in line 47 and it works fine, right? I think we have a similar test that covers this here: https://github.com/ent/ent/blob/master/entc/integration/ent/schema/dir/dir.go#L7, because the reflection package path does not aware of modules.

Reply:
Hello again, I'm not sure if I understood your message (could not find the line 47), but I have checked and the problem is present without the modules, so I have made changes to keep this PR simpler (and without new dependencies).

Please, see if you this version is better, on my side I think it is, it still have the same objectives but uses less lines of code.